.github/workflows/deploy_cms.yaml

```
@@ -0,0 +1,74 @@
+name: Deploy CMS
+
+on:
+  push:
+        branches: 'main'
+        # Run only if there are at least one change matching the following paths
+        paths:
+          - 'apps/strapi-cms/*'
```

```suggestion
          - 'apps/strapi-cms/**'
```
`*` is stricter than `**`. Ref: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet